spec/index.html

```diff
@@ -607,9 +608,9 @@ <h3>RDF Blank Nodes</h3>
       <li>The character <code>.</code> may appear anywhere except the first or last character.</li>
       <li>The characters
         <code>-</code>,
```

Comment:
I think the changes I suggested in that other document should also be applied here.